### DISCUSSION TITLE
tt5srv.log is not working

### DISCUSSION THREAD

**Discussion by farshad-fallah** [BearWare/TeamTalk5#2258] [General]
hi hello   tt5srv.log is not working  
I mean for exam  when I type .. tail -f tt5srv.log 
It shows for the previous week
 Something is always constant

**Comment by bear101** [comment]
Maybe your server has run out of disk space or the location is not writable.

**Reply by farshad-fallah** [reply]
yes   thanks my  storage was full 
my prob has fix
